drivers/serial/Kconfig.realtek_rts5912

```diff
@@ -14,7 +14,7 @@ config UART_RTS5912
 
 config UART_RTS5912_INIT_PRIORITY
 	int "RTS5912 UART wrapper init priority"
-	default 49
+	default 51
```

Comment:
Because you've changed `UART_RTS5912_INIT_PRIORITY` here, do you still need to switch the rts5912_gpio driver init priority?